projects/BFDR/NatalityRecord_constructors.cs

```diff
@@ -17,7 +17,7 @@ namespace BFDR
     public abstract partial class NatalityRecord : VitalRecord
     {
         /// <summary>Default constructor that creates a new, empty NatalityRecord.</summary>
-        public NatalityRecord(string bundleProfile) : base()
+        protected NatalityRecord(string bundleProfile) : base()
```

Comment:
what's the reasoning for changing these from public to protected? NatalityRecord is a base class but I wonder if there are any cases where it's useful for users to parse a generic NatalityRecord

Reply:
Since it's an abstract class it can't be instantiated anyway, right? Same for VitalRecord